index.html

```diff
@@ -102,5 +102,16 @@ <h2>Heading</h2>
             r.parentNode.insertBefore(e,r)}(window,document,'script','ga'));
             ga('create','UA-XXXXX-X','auto');ga('send','pageview');
         </script>
+        <!-- Hotjar Tracking Code for www.adorable-wolf.com -->
```

Comment:
Oh, that's really cool. I wasn't family with Hotjar, but it def seems to tackle the issue.